index.bs

```diff
@@ -1076,6 +1076,20 @@ with a [{{NoInterfaceObject}}] [=extended attribute=],
 and [=callback interfaces=] which declare [=constants=]
 must be annotated with an [{{Exposed}}] [=extended attribute=].
 
+<div algorithm>
+
+The <dfn>qualified name</dfn> of an [=interface=] |I| is defined as follows:
+
+1.  Let |interface| be the [=identifier=] of |I|.
```

Comment:
I'd encourage _not_ using any one-letter variables any more. How about "Let *identifier* be the identifier of *interface*"?